### PR TITLE
Add clarification about dashboard_url and async provisioning

### PR DESCRIPTION
**Pull request by pmorie**
Clarifies the issue raised in #170.

cc @luksa @avade

### CODE REVIEW

**Comment by avade** [comment]
An improvement on the current description 👍 

LGTM

File: spec.md [hunk 1]
```diff
@@ -395,7 +395,7 @@ For success responses, a broker may return the following fields. For error respo
 
 | Response field  |  Type | Description  |
 |---|---|---|
-|  dashboard_url | string  |  The URL of a web-based management user interface for the service instance; we refer to this as a service dashboard. The URL should contain enough information for the dashboard to identify the resource being accessed (<code>9189kdfsk0vfnku</code> in the example below). |
+|  dashboard_url | string  |  The URL of a web-based management user interface for the service instance; we refer to this as a service dashboard. The URL should contain enough information for the dashboard to identify the resource being accessed (<code>9189kdfsk0vfnku</code> in the example below).  Note: a broker that wishes to return `dashboard_url` for a service instance must return it with the initial response to the provision request, even if the service is provisioned asynchronously. |
```
Comment (duglin) [inline spec.md]: Since I'm not sure which PR will get merged first, can you s/must/MUST/ so we don't miss it.

**Comment by duglin** [comment]
Not thrilled with the current state of the spec on this topic, but agree with making it clear, so
LGTM

File: spec.md (hunk 1 above)
Comment (pmorie) [inline spec.md]: Yeah, will do.

**Comment by angarg12** [comment]
LGTM

**Comment by petereberlein** [comment]
LGTM

**Pull request merged**